.mergify.yml

```diff
@@ -2,7 +2,12 @@ queue_rules:
   - name: default
     conditions:
       - "#approved-reviews-by>=1"
-      - status-success=Travis CI - Pull Request
+      - check-success=gomod
+      - check-success=lint
+      - check-success=test
+      - check-success=build
```

Comment:
do these match `build (...)` and `test (...)` ?

Reply:
According to this, the job name should work - https://docs.mergify.com/conditions/#github-actions
Let's try it out 🤔